run-bitcoin.sh

```diff
@@ -2,11 +2,12 @@
 
 set -e
 
-trap "docker stop bitcoind electrumx && docker rm bitcoind electrumx" EXIT
+trap "docker stop bitcoind electrumx electrs && docker rm bitcoind electrumx electrs" EXIT
 
 LOG_START='\n\e[1;36m' # new line + bold + color
 LOG_END='\n\e[0m' # new line + reset color
 
-printf "${LOG_START}Starting bitcoind and electrumX...${LOG_END}"
+printf "${LOG_START}Starting bitcoind, electrs, and electrumX...${LOG_END}"
+printf "${LOG_START}If the build step fails, try increasing the memory allocated to docker.${LOG_END}"
 
-docker-compose -f ./bitcoin/docker-compose.yml up
\ No newline at end of file
+docker-compose -f ./bitcoin/docker-compose.yml up
```

Comment:
Looks like this command doesn't rebuild the container after  `Dockerfile` was updated and the image already exists. Thoughts around adding `--build` flag? 

Ref: https://stackoverflow.com/a/39988980/2024713